resources/views/user-settings.blade.php

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<!-- to do: CSS styling -->
+<!-- this will probably need to be moved somewhere else -->
+<style>
+    .form-popup
+    {
+        display: none;
+    }
+</style>
+</head>
+<body>
+<h1> Settings </h1>
+
+<!-- buttons to display and change audio levels/guided use toggles -->
+<button onclick="openGuidedUseForm()">Settings</button>
+<button onclick="openAudioForm()">Audio</button>
+
+<!-- forms which allow the user to see and adjust their settings -->
+<div class="form-popup" id = "GuidedUseForm"> 
+<form action="/user-settings" method="POST">
+    @csrf
+    <h2>Guided Use</h2>
+    <input type="radio" name="guided_use" value="1" {{ $guided_use_toggle ?  "checked" : ""}}>
+    <label>Guided Use: On</label>
+    <br>
+    <input type="radio" name="guided_use" value="0" <?php if ($guided_use_toggle == 0) {echo 'checked="checked"';}?> >
+    <label>Guided Use: Off</label><br>
+    <input type="submit" value="Update Guided Use Settings"><br>
+    <button onclick="closeGuidedUseForm()">Close Guided Use Settings </button>
```

Comment:
I think the `type` attribute has to be `type="button"` otherwise the button thinks it's submitting form, just noticed it when trying it in the browser
```suggestion
    <button type="button" onclick="closeGuidedUseForm()">Close Guided Use Settings </button>
```